### DISCUSSION TITLE
Keyboard shortcuts for controlling most common features

### DISCUSSION THREAD

**Discussion by kaushleshchandel** [AlexandreRouma/SDRPlusPlus#1172] [Ideas]
Hi, 

I would like to request, and also program as necessary, to add keyboard shortcuts. I am building a SDRDock that can use SDR++ and use macro ( custom buttons ) to control the SDR++.

For example a VFO, squelch control, and gain control.

Thanks,
Kaushlesh

**Comment by kaushleshchandel** [comment]
CLosing as suplciate

**Reply by FrequencyWatcher** [reply]
Do You Still Are Looking For The Answer? 
Im Trying To Figure That Out Myself And I Learned That In The Software There Is DEBUG Tab, In To That Tab You Need To ClickOn To SHOW DEMO WINDOW.  From There You Can ConfigurateThe SDR++ As You Wish And More..

**Reply by tylr-r** [reply]
@kaushleshchandel Please link the duplicate conversation in this thread so we can find it. Thanks.
